data/mods/Aftershock/npcs/prepnet_phyle/prepnet_trade.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "type": "item_group",
+    "id": "prepnet_orchard_trade",
+    "subtype": "distribution",
+    "entries": [
+      { "group": "irradiated_food", "prob": 100 },
+      { "group": "hardware", "prob": 100 },
+      { "group": "gear_survival", "prob": 100 },
+      { "group": "rad_gear", "prob": 100 },
+      { "group": "prepnet_basic_robots", "prob": 100 },
+      { "group": "prepnet_basic_grenades", "prob": 100 },
+      { "group": "ammo_any_batteries_full", "prob": 100 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "prepnet_id_cards",
```

Comment:
It looks like you are not using this item group.

Reply:
I had originally put it in then decided that the orchard prepnetter didn't need to have that.  So I left it in for when I get to larger trading posts but if you think I should remove it until then let me know.

Reply:
Eh I'd tell you to just add it to the current trade list in the meanwhile. I somewhat dislike the idea of keeping orphaned content around.